nuxt/nuxt.config.ts

```diff
@@ -53,6 +54,12 @@ export default defineNuxtConfig({
         !process.env.NUXT_PUBLIC_STACK_DOMAIN
           ? 'maev.si'
           : undefined,
+      turnstile: {
+        siteKey: '1x00000000000000000000AA', // set via environment variable `NUXT_PUBLIC_TURNSTILE_SITE_KEY` only
```

Comment:
```suggestion
        siteKey: '1x00000000000000000000AA',
```
not empty, so not set via environment variable only